docs/en/index.md

```diff
@@ -1,9 +1,13 @@
 ---
 layout: page
-title: User Sync Documentation
+title: About User Sync
+nav_link: About User Sync
+nav_level: 1
+nav_order: 1
+lang: en
 ---
 
-Welcome to the documentation center for [User Sync from Adobe](https://github.com/adobe-apiplatform/user-sync.py/releases/latest).
+Welcome to the documentation center for [User Sync from Adobe](https://github.com/adobe-apiplatform/user-sync.py).
```

Comment:
This is the right (and only) place to put the version info. Rather than having a line in the User Manual, and one in the Success Guide, just add a paragraph here that says:

Documentation Version: 2.1.1, released 2017-06-09